caps/views.py

```diff
@@ -95,8 +100,17 @@ def get_context_data(self, **kwargs):
             context["latest_year_per_km2_emissions"] = latest_year_per_km2_emissions
             context["latest_year_total_emissions"] = latest_year_total_emissions
         except DataPoint.DoesNotExist:
+            no_emissions_data = True
             context["no_emissions_data"] = True
 
+        if not no_emissions_data:
```

Comment:
I think this would be a bit clearer if this was `emissions_data` and the assignment earlier were reversed (i.e. defaults to True and False in exception.

Reply:
This happened